chronolog.py

```diff
@@ -0,0 +1,88 @@
+import os.path
+import regex as re
+
+import ChronologBackend.nlp as nlp
+import argparse
+from argparse import RawTextHelpFormatter
+
+DEFAULT_SPLITS = 10
+
+NUMERAL_PATTERN = re.compile(
```

Comment:
Might be useful to have some comments explaining what these regexes do, i.e what kind of patterns do they match.